backend/infrahub/core/constants/__init__.py

```diff
@@ -69,9 +69,11 @@ class PermissionAction(InfrahubStringEnum):
     VIEW = "view"
 
 
-class PermissionDecision(InfrahubStringEnum):
-    ALLOW = "allow"
-    DENY = "deny"
+class PermissionDecision(InfrahubNumberEnum):
+    DENY = 1
+    ALLOWED_DEFAULT = 2
+    ALLOWED_OTHER = 4
+    ALLOWED_ALL = 6
```

Comment:
Sorry I realize this is probably my fault as I typed this somewhere. but perhaps we should change it to:

```python
class PermissionDecision(InfrahubNumberEnum):
    DENY = 1
    ALLOW_DEFAULT = 2
    ALLOW_OTHER = 4
    ALLOW_ALL = 6
```

What do you think?